src/pages/api/update-manual-review.js

```diff
@@ -9,12 +9,13 @@ const logger = log4js.getLogger();
 const { Sentry } = require("../../../utils/sentry");
 
 //function for update repository
-const updateRepo = async (repoId, updatedAt) => {
+const updateRepo = async (repoId, updatedAt, comment) => {
   const updateRepo = await Repositories.update(
     {
       manual_review: false,
       review: "approved",
```

Comment:
These type of string values should be in single constants file and should be referred from there itself.

Reply:
i have done this changes in another Pr